cf-deploy-teardown/cf-deploy-teardown.yml

```diff
@@ -0,0 +1,43 @@
+# Concourse pipeline to build deploy and teardown cf against a kube cluster
+groups: []
+
+resources:
+- name: cf-ci
+  type: git
+  source:
+    uri: https://github.com/SUSE/cf-ci.git
+    branch: master
+    paths:
+    - cf-deploy-teardown/*
+
+- name: s3.scf-kube-yml
+  type: s3
+  source:
+    access_key_id: {{s3-access-key}}
+    bucket: kube-dist
+    endpoint: {{s3-endpoint}}
+    regexp: scf-kube-(.*).zip$
+    secret_access_key: {{s3-secret-key}}
+    private: true
+
+jobs:
+- name: cf-deploy-teardown
+  plan:
+  - aggregate:
+
+    - get: cf-ci
+      trigger: true
+
+    - get: s3.scf-kube-yml
+      trigger: true
+
+  - task: cf-deploy
+    file: cf-ci/cf-deploy-teardown/tasks/cf-deploy.yml
+    params:
+      K8S_HOST_IP: {{k8s-host-ip}}
+      K8S_HOSTNAME: {{k8s-hostname}}
+      K8S_USER: {{k8s-host-user}}
+      K8S_PASSWORD: {{k8s-host-password}}
+      K8S_HOST_PORT: {{k8s-host-port}}
```

Comment:
I dont know how thats going to land. For now I am assuming that we have a k8s cluster for this pipeline, and its values are in secrets file in SUSE/cloudfoundry. I guess once we have a pooled resource of some kind we can work on a seprate ticket: test pipeline to use k8 pool as resource instead of k8s details hardcoded in secrets file